node/tests/TestUtilities.ts

```diff
@@ -689,8 +689,6 @@ export async function transactionTest(
     responseData.push(["pubsubChannels()", []]);
     baseTransaction.pubsubNumPat();
     responseData.push(["pubsubNumPat()", 0]);
-    baseTransaction.pubsubNumSub();
```

Comment:
You can keep `baseTransaction.pubsubNumSub([]);`

Reply:
I removed `Optional` channel in Transaction. It will fail if I revert this changes.